i18n/zh/docusaurus-plugin-content-docs/current/architecture/edge/metamanager.md

```diff
@@ -2,22 +2,22 @@
 title: MetaManager
 sidebar_position: 5
 ---
-## Overview
-MetaManager is the message processor between edged and edgehub.
-It's also responsible for storing/retrieving metadata to/from a lightweight database(SQLite).
-
-Metamanager receives different types of messages based on the operations listed below :
-  - Insert
-  - Update
-  - Delete
-  - Query
-  - Response
-  - NodeConnection
-  - MetaSync
-
-## Insert Operation
+## 概述
+MetaManager 是 edged 和 edgehub 之间的消息处理器。它还负责在轻量级数据库（SQLite）中存储/检索元数据。
+
+Metamanager 有下列操作：
+  - Insert：插入
+  - Update：更新
+  - Delete：删除
+  - Query：查询
+  - Response：响应
+  - NodeConnection：节点连接
+  - MetaSync：Meta 同步
+
+## Insert：插入
 `Insert` operation messages are received via the cloud when new objects are created.
 An example could be a new user application pod created/deployed through the cloud.
```

Comment:
need delete the english part

Reply:
I'm not quite clear about your reason. This is a common operation for MetaManager. Are you saying that the part about MetaManager in KubeEdge has been removed from the architecture?

Reply:
Thank you for your advice, I have made the corresponding changes.